config.go

```diff
@@ -722,9 +682,8 @@ func loadConfig() (*config, []string, error) {
 
 	// Load dcrd RPC certificate.
 	if !fileExists(cfg.DcrdRPCCert) {
-		str := "%s: dcrd RPC certificate (%v) not found"
-		err := fmt.Errorf(str, funcName, cfg.DcrdRPCCert)
-		fmt.Fprintln(os.Stderr, err)
+		err := fmt.Errorf("%s: dcrd RPC certificate (%v) not found", funcName,
+			cfg.DcrdRPCCert)
 		return nil, nil, err
```

Comment:
This wasnt showing usage previously, now it is.

Reply:
Suppressed.